dev-tools/cherrypick_pr

```diff
@@ -135,6 +135,9 @@ def main():
         session.post(
             base + "/issues/{}/labels".format(new_pr["number"]), json=["backport", "review"])
 
+        # remove needs backport label from the original PR
+        session.delete(base + "/issues/{}/labels/needs_backport".format(args.pr_number))
```

Comment:
My only two concern with this is that the label should only be removed, when the PR is merged. And it also does not work if their are backports needed to 5.3 and 5.4